.github/workflows/windows_installer.yml

```diff
@@ -0,0 +1,61 @@
+name: Create Windows Installer
+
+on:
+  workflow_dispatch:
+  push:
+    tags: [ v* ]
+  pull_request:
+
+jobs:
+  build:
+    runs-on: windows-2019
+    steps:
+      - uses: actions/checkout@v3
+      - name: System Info
+        run: |
+          systeminfo || true
+          env
+      - name: Set up Python 3.x
+        uses: actions/setup-python@v4
+        with:
+          # Semantic version range syntax or exact version of a Python version
+          python-version: '3.11'
+          # Optional - x64 or x86 architecture, defaults to x64
+          architecture: 'x64'
+          cache: 'pip'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -e .
+      - name: Freeze dependencies to requirements.txt
+        run: |
+          pip freeze > ${{ github.workspace }}/requirements.txt
+      - name: Download PyApp
+        run: |
+          curl -L -o pyapp.zip "https://github.com/ofek/pyapp/releases/download/v0.12.0/source.zip"
+      - name: Unzip PyApp
+        run: |
+          unzip pyapp.zip
+        shell: bash
+      - name: List Directory Contents
+        run: |
+          ls -la
+        shell: bash
+      - name: Create Windows Installer
+        run: |
+          WORKSPACE=$(cygpath -u "${{ github.workspace}}")
+          cd $WORKSPACE/pyapp-v0.12.0
+          export PYAPP_PROJECT_NAME=freemocap
+          export PYAPP_PROJECT_VERSION=1.0.33
+          export PYAPP_PROJECT_DEPENDENCY_FILE=$WORKSPACE/requirements.txt
+          export PYAPP_EXEC_SCRIPT=$WORKSPACE/freemocap/__main__.py
+          cargo build --release
+          cargo install pyapp --force --root $WORKSPACE
+          cd $WORKSPACE
+          mv $WORKSPACE/bin/pyapp freemocap_app && chmod +x freemocap_app
+          tar -czvf freemocap_app.tar.gz freemocap_app
```

Comment:
Windows Installer creates a `.tar.gz` file - This like looks similar to the Mac/Linux builders. Do you know how to fix?

Reply:
Yep, the reason for that is here: https://github.com/actions/upload-artifact 
"Permission Loss
File permissions are not maintained during artifact upload. All directories will have 755 and all files will have 644. For example, if you make a file executable using chmod and then upload that file, post-download the file is no longer guaranteed to be set as an executable.

If you must preserve permissions, you can tar all of your files together before artifact upload. Post download, the tar file will maintain file permissions and case sensitivity."
So you should be able to use an upgraded zip manager like winRAR or 7Zip to unzip it. I'm not able to test it on windows, but we may have success just changing the file extension and uploading it as an `.exe` though, I'll try it out and ping you when it's uploaded and ready to test.

Reply:
@jonmatthis the .exe build just finished, you should be download and check if it is able to run: https://github.com/freemocap/freemocap/actions/runs/7388774233